src/Markdig/Helpers/CharacterMap.cs

```diff
@@ -127,7 +123,7 @@ public int IndexOfOpeningCharacter(string text, int start, int end)
                         for (int i = start; i <= end; i++)
                         {
                             var c = pText[i];
-                            if (c < 128 && openingChars[c])
+                            if (c < 128 && isOpeningCharacter[c])
```

Comment:
I don't like that we're now fetching a field every time. We should try to see if the first half of this PR (ctor change) fixes the UWP issue